draco/analysis/delay.py

```diff
@@ -999,7 +1156,14 @@ def _draw_ps_sample(d):
         d_samp = _draw_signal_sample(S_samp)
         S_samp = _draw_ps_sample(d_samp)
 
-        spec.append(S_samp)
+        # If we're dealing with a complex delay spectrum, S_samp will be the power
+        # spectrum of the (alternating) real and imaginary delay spectrum components,
+        # so we need to compress S_samp down to the power spectrum of the complex
+        # elements
+        if complex_input:
+            spec.append(S_samp[0::2] + S_samp[1::2])
+        else:
+            spec.append(S_samp)
```

Comment:
I am a bit unclear about whether this is quite what we want. $\DeclareMathOperator{\var}{var}$

We are calculating delay power spectra, and the underlying assumption is that the underlying statistics are circularly symmetric complex Gaussian (so $D = \var(d_\text{real}) + \var(d_\text{imag})$ ), and that's what we output in the end. However, in the middle we calculate the separate real and imaginary PS samples, and then feed that back into the drawing of the signal sample. That means that the delay noise perturbation which are added into the signal sample do not have circularly symmetric statistics.

Ultimately I'm trying to understand why the output seems "noisier", and while I don't quite know what effect this is going to have with respect to that, it's a bit weird.